build.js

```diff
@@ -31,6 +31,10 @@ var child_process = require('child_process');
 
 var useCompilers = String(process.argv[2]).toLowerCase() === "compilers";
 
+var isOptional = function isOptional(category) {
```

Comment:
Just to avoid hoisting - if you have a strong preference I'll be happy to change it to a declaration.